readthedocs/doc_builder/python_environments.py

```diff
@@ -383,6 +390,7 @@ def install_core_requirements(self):
             ])
 
         cmd = copy.copy(pip_install_cmd)
+        cmd.extend(self._pip_extra_args())
```

Comment:
Why don't we add it in line 344, next to cache, so they all are together?

```
            *self._pip_cache_cmd_argument(),
```

Reply:
So, in 334 we are installing pip itself, so the previous version of pip may not support the `--use-feature` option.